packages/crayons-core/src/components/data-table/custom-cells/anchor/custom-cell-anchor.scss

```diff
@@ -0,0 +1,5 @@
+.anchor {
```

Comment:
Move this as a Global class/mixin inside typography? 
I think anchors might be the same across the application!

Reply:
Moved this to mixins and exposed a css class for anchor.